### ISSUE TITLE
Is this the same as the easyerp online version?

### ISSUE THREAD

**Issue by woakes070048** [EasyERP/EasyERP_open_source#11]
If this is the same as a online version my team and i would love to collaborate with you guys to help make the software even better. we have been looking for something like this in this space. a little about is at pegas.io

**Comment by jacksolutions** [comment]
even we are ready to take it further, so if the original author has ditched this project, i would like to start from where they left and take it further, if you guys are also ready to collaborate, let me know @woakes070048

**Comment by woakes070048** [comment]
@jacksolutions we have been using erpnext for our projects but I am will in look st this project again

**Comment by jacksolutions** [comment]
i know erpnext but 

1) it is python
2) its a resource hogging, it is even crashing on 4 GB

**Comment by jacksolutions** [comment]
easyerp is easy on resource and perfectly suited for SMEs. do you have any other nodejs based ERP recommendation which is upto date? the only thing i dont like about easyerp is backbone.js

**Comment by woakes070048** [comment]
I don't off the top of my head. Erpnext and easyerp differnt animals. Are you looking to develop for your internal use or for customers?

**Comment by jacksolutions** [comment]
internal use as well as for my customers

**Comment by sm2x** [comment]
yes jack, lets do it, nice to have an ERP that is based on nodejs and nosql

**Comment by woakes070048** [comment]
@sm2x @jacksolutions do you guys have an email so we can invite you guys to a slack channel?

**Comment by jacksolutions** [comment]
let me know once you invite so that i can remove my email

**Comment by woakes070048** [comment]
Okay just sent the invite

**Comment by jacksolutions** [comment]
okay i have got the invitee, thanks

**Comment by woakes070048** [comment]
Great

**Comment by onamission** [comment]
I am also interested. How can I get involved?

**Comment by proffalken** [comment]
@onamission checkout #16 - we're discussing how this might look in future over there.

**Comment by kanishkpuri1987** [comment]
I am also interested.
